scenarios/07_Ally_From_the_Past.cfg

```diff
@@ -84,6 +84,9 @@
 #ifdef EASY
                 {TRAIT_RESILIENT}
 #endif
+#ifdef NIGHTMARE
```

Comment:
And same thing here.

Reply:
[086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).